tests/kernel/run_ltp.pm

```diff
@@ -262,6 +262,30 @@ sub run {
     script_run('$LTPROOT/ver_linux');
 
     if ($is_network) {
+        # poo#18762: Sometimes there is physical NIC which is not configured.
+        # One of the reasons can be renaming by udev rule in
+        # /etc/udev/rules.d/70-persistent-net.rules. This breaks some tests
+        # (even net namespaces based ones).
+        # Workaround: configure physical NIS (if needed) and "flush" network
+        # settings with upload_logs().
+        my $tmp = << 'EOF';
+dir=/sys/class/net
+ifaces="`basename -a $dir/* | grep -v -e ^lo -e ^tun -e ^virbr -e ^vnet`"
+for iface in $ifaces; do
+    config=/etc/sysconfig/network/ifcfg-$iface
+    if [ `cat $dir/$iface/operstate` = "down" ] && [ ! -e $config ]; then
+        echo "WARNING: create config '$config'"
+        printf "BOOTPROTO='dhcp'\nSTARTMODE='auto'\nDHCLIENT_SET_DEFAULT_ROUTE='yes'\n" > $config
+        systemctl restart network
+    fi
+done
+EOF
+        script_run($tmp);
+        $tmp = '/root/workaround-empty-file.txt';
+        script_run("touch $tmp");
+        # deliberately send after setting net to fix setup
+        upload_logs($tmp);
```

Comment:
How does it flush the network settings; is it just because it creates a TCP connection to the outside world? Maybe you could just download a file to /dev/null from the host?

Possibly we should not fix this in OpenQA or LTP. I will comment on the progress issue.